### ISSUE TITLE
Problem with existing fcm config file

### ISSUE THREAD

**Issue by pwFoo** [tinode/chat#339]
Hi,
after some time I updated tinode to latest version and fcm push notifications stopped working without changes for fcm in tinode container. Maybe there is a tinode / fcm breaking change I don't know?

I searched https://console.firebase.google.com/ (project -> settings -> my apps -> sdk config snippet, no file download found) for a maybe changed config file and it looks like syntax is changed completely? 

"Old" file:
```
{
  "type": "service_account",
  "project_id": [...]
```

File I found online:
```
const firebaseConfig = {
  apiKey: [...]
```

Haven't found a file like the old one online searched the firebase project...
Could you help me how to change fcm settings to get it working again?

P.S.
That's why I asked for moving away from fcm. With webpush-lib there is no need to manage a FCM project and config file. Just to generate keys for the server side.

**Comment by pwFoo** [comment]
Error message:
```
Failed to initialize push notifications FirebaseError: Installations: Missing App configuration value: "projectId" (installations/missing-app-config-values).
```

**Comment by or-else** [comment]
See here: https://github.com/tinode/webapp/commit/6fee1ce5ea27ae4cbc946908bafc0116818619d3

**Comment by pwFoo** [comment]
Hi, thanks. That was was I missed during I stopped testing tinode...

The file is in static/ directory which isn't a volume. Is it possible to configure by ENV or move that file to a docker volume?

Manually replaced the content but get a new error message.
```
Failed to initialize push notifications FirebaseError: Messaging: The public VAPID key must be a string. (messaging/invalid-public-vapid-key).
```

Shoud be a string configured by ENV...
```
      FCM_CRED_FILE: "/fcm/<JSON_FILE>.json"
      FCM_SENDER_ID: "<ID>"
      FCM_VAPID_KEY: "<VAPID_KEY>"
```
Tested with and without `"`.

**Comment by or-else** [comment]
I've released v0.16.3-rc6. It should take care of this problem.

**Comment by or-else** [comment]
@pwFoo Have you  tried the v0.16.3-rc6?

**Comment by pwFoo** [comment]
Hi, still problems with service worker (mobile device) and push initialization (desktop)
```
Failed to initialize push notifications FirebaseError: Messaging: The public VAPID key must be a string. (messaging/invalid-public-vapid-key).
```

**Comment by or-else** [comment]
Have you followed the new instructions?
https://github.com/tinode/chat/tree/devel/docker#supported-environment-variables

**Comment by or-else** [comment]
Unless I get more feedback on this issue, I'll assume that the issue is resolved and will release the 0.16.3

**Comment by pwFoo** [comment]
Hi, updated to 0.16.3 today and looks good. I was able to activate notifications with the browser. 
Need to test notifications with some friends. 
I close the issue. Thanks!
